dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/services/QaReportEventListenerService.kt

```diff
@@ -73,6 +77,8 @@ class QaReportEventListenerService(
         messageUtils.rejectMessageOnException {
             if (actionType == ActionType.DeleteData) {
                 reportManager.deleteAllQaReportsForDataId(dataId)
+                reviewQueueRepository.deleteByDataId(dataId)
+                reviewHistoryRepository.deleteByDataId(dataId)
```

Comment:
Hier war ich mir nicht ganz sicher:

Das ist ja eine history-Tabelle.
An einer history schraubt man ja normalerweise nicht rum, auch wenn es das betroffene dataset nicht mehr gibt.
Sollten wir die history nicht unangetastet lassen?

Reply:
Fairer Punkt. Aber die QA Reports sind doch eigentlich auch eine Historie. Plus es stehen einfach Einträge dann in der Tabelle mit denen man nichts mehr anfangen kann, da ja auch alle meta informationen zu der dataId gelöscht werden.